### PR TITLE
minor fix in tests/stdlib/tbitops.nim.cfg

### PR DESCRIPTION
**Pull request by rockcavera**

### CODE REVIEW

File: tests/stdlib/tbitops.nim.cfg [hunk 1]
```diff
@@ -1 +1 @@
--d:noUndefinedBitOps
+-d:noUndefinedBitOpts
```
Comment (Araq) [inline tests/stdlib/tbitops.nim.cfg:1]: What? This should be noUndefinedBitOps somewhere else.

Comment (rockcavera) [reply tests/stdlib/tbitops.nim.cfg:1]: I thought it was wrong here, considering that in the `bitops` documentation and in the codes everyone is like` noUndefinedBitOpts`.

Comment (rockcavera) [reply tests/stdlib/tbitops.nim.cfg:1]: So the only correct place is in this .cfg.

Because even `noIntrinsicsBitOpts` is Opts.

Comment (rockcavera) [reply tests/stdlib/tbitops.nim.cfg:1]: Now, which one would be correct?

- no option of bit intrinsics?
- no intrinsic bitops (bit operations)?

Comment (Araq) [reply tests/stdlib/tbitops.nim.cfg:1]: My mistake.

**Pull request merged**